backend/src/common/dto/users.dto.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNumberString } from 'class-validator';
+
+export class UserCreateReqDto {
+  name: string;
```

Comment:
試しに/userにPOSTしてみるとエラーになりました。

以下のように修正するとうまく動きました。
もしかしたら、全部のクラスに対して、なにかしらバリデータ設定しないといけないみたいです。
https://github.com/typestack/class-validator#validation-decorators

```suggestion
export class UserCreateReqDto {
  @IsString()
  name: string;
```

Reply:
本チケットの対応でも別チケット対応でも大丈夫です！